data/ships.txt

```diff
@@ -2685,7 +2681,7 @@ ship "Star Barge"
 		
 	engine -9 38
 	engine 9 38
-	turret 0 -8 "Anti-Missile Turret"
+	turret 0 -8
```

Comment:
no idea why ya did this, adding that it still has the AM turret installed, ya just removed it from the hardpoint

Reply:
Look at the diff. The change you're looking at is obsolete.

Reply:
ah a lot of reverts, that explains it lol